### PR TITLE
Completion feature for Hegel Playground (draft)

### PR DESCRIPTION
**Pull request by YevhenKap**
I have wrote simple completion feature for Monaco Editor. But I have problem with testing it. Locally it doesn't work, though code is created according to [official API](https://microsoft.github.io/monaco-editor/api/index.html). It must work... I spend several days for finding problem, but despair is followed by me. Can anyone give me advise? Maybe there is stupid mistake?

### CODE REVIEW

**Comment by YevhenKap** [comment]
I have checked `getCompletionItems` function returns exactly what we need, but values aren't showed in editor(

**Comment by JSMonk** [comment]
I will try to reproduce it.
Thank you a lot.
You are awesome ^_^

File: packages/docs/src/docs/try/components/hegel.js [hunk 1]
```diff
@@ -100,6 +107,108 @@ export async function getDiagnostics(sourceCode) {
   return errors.map(toTransferableObject);
 }
 
+/**
+ * Summon and format completion items.
+ * @param {import("monaco-editor").languages.CompletionItemKind} completionItemKind
```
Comment (Raynos) [inline packages/docs/src/docs/try/components/hegel.js:112]: Is this supported by `hegel` itself :heart_eyes: or is this typescript syntax ?

Comment (YevhenKap) [reply packages/docs/src/docs/try/components/hegel.js:112]: At current moment Hegel does not support comments. It is just for TypeScript IntelliSense.

**Comment by YevhenKap** [comment]
I have opened [issue](https://github.com/microsoft/monaco-editor/issues/1976).

**Pull request merged**